service-front/module/Application/src/Controller/DonorFlowController.php

```diff
@@ -95,11 +95,15 @@ public function donorDetailsMatchCheckAction(): ViewModel
         $detailsData['formatted_dob'] = (new \DateTime($detailsData['dob']))->format("d F Y");
 
         $view = new ViewModel();
+        /** @var string $base */
+        $base = getenv("SIRIUS_BASE_URL");
```

Comment:
We don't like using `getenv` in application code because it creates a configuration injection dependency and makes it hard to mock. Instead, call `getenv` in the factory and pass through the value as a constructor variable (e.g. `public function __construct(private readonly string $siriusBaseUrl`)